test/httpd-world_test.coffee

```diff
@@ -4,7 +4,7 @@ http = require('http')
 
 expect = require('chai').expect
 
-process.env.EXPRESS_PORT = 8080
+process.env.EXPRESS_PORT = 8081
```

Comment:
It seems that we should find unused ports :sweat_smile: 
I'll fix about the port on another PR.

Could you revert this change and force push :question: